src/server/CodeStyle/CodeStyle.py

```
@@ -19,12 +20,145 @@ def parse_java_code(code):
 
     return tree, tokens
 
-def clean_code(code):
-    cleaned_code = re.sub(r'[\t\n\r]+', '', code)
-    cleaned_code = re.sub(r' {2,}', ' ', cleaned_code)
-    cleaned_code = re.sub(r'^ +', '', cleaned_code, flags=re.M)
+def clean_code(code, configs):
```

Pylint (3.3.7)

[refactor] Too many local variables (25/15)

(R0914)